pkg/controllers/provisioning/v1alpha1/reallocation/utilization.go

```diff
@@ -32,24 +32,6 @@ type Utilization struct {
 	kubeClient client.Client
 }
 
-func (u *Utilization) Reconcile(ctx context.Context, provisioner *v1alpha1.Provisioner) error {
```

Comment:
Where'd this logic go? Is this implemented somewhere else not in this PR?

Reply:
I accidentally left this replicated in the previous PR. I removed this as it wasn't being used.